custom_components/tesla_custom/util.py

```diff
@@ -9,3 +9,15 @@
     from homeassistant.util.ssl import client_context
 
     SSL_CONTEXT = client_context()
+
+
+def safeget(dct, *keys, default=None):
```

Comment:
Since you're trying to add this in teslajsonpy, I'm not a fan of defining it twice. Let's get it in at the library level and then access it.

Reply:
not too keen to reuse it from teslajsonpy, as its just a helper function.
you could copy that code and run it in any python script (i use it in mine quite a bit).

if it was a bit of code specific to our tesla code, I'd agree with you, but its just a dictionary helper

Reply:
If it ends up having a bug or getting deprecated, we'll have to fix it in two places. There's no point in creating more work when you already want it in the underlying library.